binarySearch/binarySearch.js

```diff
@@ -12,4 +12,22 @@
 
 const binarySearch = (nums, target) => {
 
+	let min = 0;
+    let max = nums.length - 1;
+
+    while (min <= max){
+
```

Comment:
I know it makes it nice for you to read, but maybe being more consistent with spaces here would be nice. I could see a dev reviewing this code go either way on the subject. but if you have an empty space between code blocks most linters will be mad at you :)